src/indexer/stamper.rs

```diff
@@ -101,6 +101,7 @@ mod test {
 
     use super::Stamper;
 
+    #[allow(clippy::redundant_clone)]
```

Comment:
Yeah... that's a bug of clippy isn't it?

Reply:
Yes, the lint does not consider interior mutability.